src/js/gui.js

```diff
@@ -355,7 +355,7 @@ class GuiControl {
     }
     selectDefaultTabWhenConnected() {
         const result = getConfig(['rememberLastTab', 'lastTab']);
-        const tab = result.rememberLastTab && result.lastTab ? result.lastTab : 'tab_setup';
+        const tab = result.rememberLastTab && result.lastTab && this.allowedTabs.indexOf(result.lastTab.substring(4)) !== -1 ? result.lastTab : 'tab_setup';
```

Comment:
Would something liek this work?
```suggestion
        const tab = result.rememberLastTab && result.lastTab && this.allowedTabs.includes(result.lastTab.substring(4)) ? result.lastTab : 'tab_setup';
```

Reply:
:grin: Thanks :+1: